### DISCUSSION TITLE
Create a function column

### DISCUSSION THREAD

**Discussion by oswaldB** [directus/directus#3859] [Feature Requests]
It would be awesome if we can use a function column that apply a js function to a specific column like airtable function column.

**Comment by nelolka** [comment]
You can do that with a [Custom API Hook](https://docs.directus.io/guides/extensions/api-hooks.html)

**Reply by benhaynes** [reply]
Would be cool to have it configurable in the UI too!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
